cime_config/usermods_dirs/NEON/defaults/shell_commands

```diff
@@ -1,16 +1,26 @@
+#!/bin/bash
 ./xmlchange CLM_USRDAT_NAME=NEON
-./xmlchange RUN_STARTDATE=2018-01-01
-./xmlchange CLM_NML_USE_CASE=1850-2100_SSP3-7.0_transient
 ./xmlchange CCSM_CO2_PPMV=408.83
+# Set data forcing data to future scenario so will have data from 2018 to present-day
 ./xmlchange DATM_PRESAERO=SSP3-7.0
 ./xmlchange DATM_PRESNDEP=SSP3-7.0
 ./xmlchange DATM_PRESO3=SSP3-7.0
+# Explicitly set the MPI library to mpi-serial so won't have the build/run complexity of a full MPI library
+./xmlchange MPILIB=mpi-serial
+# Set years to run forcing data over
 ./xmlchange DATM_YR_ALIGN=2018,DATM_YR_END=2021,DATM_YR_START=2018
+compset=`./xmlquery COMPSET --value`
+# For a transient case run the whole length and don't cycle
+if  [[ $compset =~ ^HIST ]]; then
+  ./xmlchange DATM_YR_END=2022
+  ./xmlchange RUN_STARTDATE=2018-01-01
+  # Number of months that can be run for the full transient case
+  ./xmlchange STOP_OPTION="nmonths"
```

Comment:
This seems to be written over, I'm assuming by run_neon?

```
NEON_testV2/BLAN.transient> ./xmlquery STOP_OPTION,STOP_N

Results in group run_begin_stop_restart
	STOP_OPTION: ndays
	STOP_N: 51
```

I'll hard code this to be handled correctly in run_neon

Reply:
We've handled this.